.github/workflows/check_interface_compatibility.yml

```diff
@@ -0,0 +1,80 @@
+name: Check interface compatibility
+
+on:
+  pull_request_target:
+
+permissions:
+  pull-requests: write
+
+jobs:
+  check-interface-compatibility:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
For symmetry with the other checkout, a `name` here describing what it checks out would be nice.

Reply:
updated